styles/style.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Barlow:wght@200&display=swap');
+@font-face {
+    font-family: Norwester;
+    src: url(../assets/norwester.woff);
+}
+
+:root{
+    --green0: #2e4530;
+    --green1: #4e6647;
+    --brown0: #ba8559;
+    --brown1: #7e5736;
+    --taupe0: #bfb7b7;
+}
+
+a:link{
+    color: #4e6647;
+}
+a:hover{
+    color:#bfb7b7;
+}
+a:visited{
+    color: #ba8559;
+}
+
+a{
+    font-family: "Barlow", sans-serif;
+    padding: 20px;
```

Comment:
Having padding around all `a` elements is likely to cause problems in the general case. Consider that `a` tags often appear in the middle of other text.

This is causing problems on the Fun Facts page, where the paddings of the nearby anchors are overlapping one another in strange ways. For instance, Facts has no link, but is covered by the Gallery link, while the Gallery link is itself covered by the blog link.

I'd suggest adding additoinal specifiers to the matching rule so that it can be targetted more effectively, and the styles affect the nav on fun facts should be re-examined.